solidity/contracts/converter/interfaces/IBancorConverter.sol

```diff
@@ -11,4 +11,6 @@ contract IBancorConverter {
     function conversionWhitelist() public view returns (IWhitelist) {}
     // deprecated, backward compatibility
     function change(IERC20Token _fromToken, IERC20Token _toToken, uint256 _amount, uint256 _minReturn) public returns (uint256);
+    function getConnectorBalance(IERC20Token _connectorToken) public view returns (uint256);
```

Comment:
1. these two aren't deprecated so they need to move up
2. getConnectorBalance will only return the balance but we might as well just add all the connector values by returning all connector values (isSet shouldn't be returned but should be checked when returning the values)
3. getFinalAmount is definitely not a proper function name for an external interface and should be internal if not private in the converter. I think it's better to return the fee and deal with the calculation externally